internal/blockprocessor/committer_test.go

```diff
@@ -845,16 +846,19 @@ func TestStateDBCommitterForDBBlock(t *testing.T) {
 	t.Parallel()
 
 	tests := []struct {
-		name              string
-		setup             func(db worldstate.DB)
-		expectedDBsBefore []string
-		dbAdminTx         *types.DBAdministrationTx
-		expectedDBsAfter  []string
-		expectedIndex     map[string]*types.DBIndex
-		valInfo           []*types.ValidationInfo
+		name                    string
+		setup                   func(db worldstate.DB)
+		expectedDBsBefore       []string
+		expectedIndexDBsBefore  []string
+		dbAdminTx               *types.DBAdministrationTx
+		expectedDBsAfter        []string
+		expectedIndex           map[string]*types.DBIndex
```

Comment:
In line with the naming convention around, `expectedIndex` sould be `expectedIndexAfter` no?

And, in addition, in the spirit of OCD, I noticed there is no `expectedIndexBefore`...

Reply:
Done.